lib/seccomp-tools/dumper.rb

```diff
@@ -129,5 +129,39 @@ def handle_child(*args)
         exit(1)
       end
     end
+
+    # Dump seccomp-bpf using ptrace of an existing process.
+    #
+    # @param [Integer] pid
+    #   Target process identifier.
+    # @param [Integer] limit
+    #   Number of filters to dump. Negative number for unlimited.
+    # @yieldparam [String] bpf
+    #   Seccomp bpf in raw bytes.
+    # @yieldparam [Symbol] arch
+    #   Architecture of the target process (always nil right now).
+    # @return [Array<Object>, Array<String>]
+    #   Return the block returned. If block is not given, array of raw bytes will be returned.
+    # @example
+    #   dump(pid1, 1)
+    #   #=> []
```

Comment:
add comment about why this is supposed to be empty (`pid1` is a process without seccomp installed or?)
Also suggest to do launch a process (through popen or Open3) and use its pid as examples here. Binaries under `spec/binary/` can be used.